hedera-node/hedera-app/src/testFixtures/java/com/hedera/node/app/fixtures/state/FakeServiceMigrator.java

```diff
@@ -49,13 +49,14 @@ public List<StateChanges.Builder> doMigrations(
             @NonNull final ServicesRegistry servicesRegistry,
             @Nullable final SoftwareVersion previousVersion,
             @NonNull final SoftwareVersion currentVersion,
-            @NonNull final Configuration config,
+            @NonNull final Configuration nodeConfiguration,
+            @Nullable final Configuration platformConfiguration,
```

Comment:
`platformConfiguration` is not passed to the `FakeSchemaRegistry`. Its not needed there, but the concern it doesn't 1:1 correspond to the real service (`MerkleSchemaRegistry`).